packages/authgear-react-native/index.d.ts

```diff
@@ -840,6 +840,9 @@ export declare interface PromoteOptions {
 export declare class ReactNativeContainer {
   private dpopProvider;
   /**
+   * Delegation for customizing the behavior of your application.
+   * You can implement your own delegation and the container will evaluate them if needed.
+   *
    * @public
    */
   delegate?: ReactNativeContainerDelegate;

```

Comment:
The description for `name` should be consistent. You have updated ConfigureOptions.name, so XXXContainer.name should share the same description.

In general, the description in XXXOptions and the description of the field should resemble.